owl-bot-staging/google-maps-fleetengine/delivery/v1/.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
```

Comment:
This change is incorrect. `owl-bot-staging` should not include the client for `google/maps/fleetengine/delivery/v1/`. Only `google/maps/fleetengine/v1/` is expected